README.md

```diff
@@ -50,12 +50,16 @@ stack test --pedantic
 
 ### Running HLint
 
-All code in this repository should be as idiomatic as possible, so it is
-highly recommended to run `hlint` on your sources before opening a
-*pull request*.
+All code in this repository should be as idiomatic as possible, so we
+enforce in [Travis-CI] that it returns `No hints` when processed by
```

Comment:
We do have the build status image, so it wouldn't strictly be necessary. You're right that we mention Travis multiple times in this file. If we want to link it I suggest we do something to only have to link to it once:

```
[e 1][ex]
[e 2][ex]

[ex]: http://example.com
```

[e 1][ex]
[e 2][ex]

[ex]: http://example.com